project/Dependencies.scala

```diff
@@ -800,9 +790,20 @@ object Dependencies {
   )
 
   private val protobufJavaOverrides = List(
-    "com.google.protobuf" % "protobuf-java" % "3.21.2",
+    "com.google.protobuf" % "protobuf-java" % "3.25.3",
   )
 
+  val servicesDependencies: List[ModuleID] = List(
+    "com.google.cloud" % "google-cloud-billing" % "2.47.0",
+    "com.google.api" % "gax-grpc" % googleGaxGrpcV,
+    "org.apache.commons" % "commons-csv" % commonsCsvV,
+    "bio.terra" % "bard-client-resttemplate" % bardClientV
+      exclude("org.springframework", "spring-aop")
+      exclude("org.springframework", "spring-jcl"),
+    "org.apache.httpcomponents.client5" % "httpclient5" % apacheHttpClient5V // Needed for rest-template connection pooling
+
+  ) ++ testDatabaseDependencies ++ akkaHttpDependencies ++ mockServerDependencies ++ googleCloudDependencies
```

Comment:
Is it feasible to break up `googleCloudDependencies` and bring only that smaller chunk into services?

Right now its value is `<a bunch of individual stuff>` ++ googleGenomicsV2Alpha1Dependency ++ googleLifeSciencesV2BetaDependency ++ googleBatchv1Dependency` and I'm pretty sure services doesn't need any of the job running stuff.